src/GraphQL.Authorization.ApiTests/BomTests.cs

```diff
@@ -0,0 +1,40 @@
+using System.Runtime.CompilerServices;
+
+namespace GraphQL.Authorization.ApiTests;
+
+public class BomTests
+{
+    // https://github.com/graphql-dotnet/graphql-dotnet/pull/3477
+    [Fact]
+    public void Files_Should_Not_Use_BOM()
+    {
+        string GetPath([CallerFilePath] string path = "") => path; // <GIT_ROOT>\src\GraphQL.ApiTests\BomTests.cs
+
+        var gitRoot = new DirectoryInfo(GetPath()).Parent!.Parent!.Parent!;
```

Comment:
This test should verify the sln exists in this folder prior to the scanning or this test should be removed.

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/3478#discussion_r1063246769

Reply:
```suggestion
        var gitRoot = new DirectoryInfo(GetPath()).Parent!.Parent!.Parent!;
        if (!File.Exists(Path.Combine(gitRoot.FullName, "src", "GraphQL.Authorization.sln")))
            throw new InvalidOperationException("Unable to find solution root");
```
Respectfully, this is my suggestion.